core/src/main/kotlin/ch/ergon/dope/resolvable/fromable/Bucket.kt

```diff
@@ -17,3 +18,11 @@ class AliasedBucket(name: String, val alias: String) : Bucket(name) {
 }
 
 fun AliasedBucket.field(field: String): Field<StringType> = Field(field, alias)
+
+fun Bucket.all(): Expression = Field<StringType>(
```

Comment:
Since you are returning an Expression you can create `SELECT RAW sync.* FROM sync` in dope, which is not valid syntax in CB

Reply:
done